deployment/network-operator/values.yaml

```diff
@@ -174,6 +175,10 @@ ofedDriver:
   # env:
   #   - name: EXAMPLE_ENV_VAR
   #     value: example_env_var_value
+  containerResources:
+    - name: "mofed-container"
```

Comment:
Do we want to include all resources here explicitly? Maybe It is better to provide a single commented example? I want to avoid situation when we will need to always update the list when adding a new container

Reply:
Do you mean a single example like this?
```
containerResources:
  - name: "mofed-container"
     // requests:
     //   "cpu": "500Mi"
     //   "memory": "300Mi"
     // limits:
     //   "cpu": "500Mi"
     //   "memory": "300Mi"
 ```